webapp/redux/middleware/utils.js

```diff
@@ -32,8 +28,7 @@ export const downloadJsonResponse = (jsonData) => {
   forceBlobDownload(blob, filename);
 };
 
-export const downloadPythonResponse = (textData) => {
-  const filename = `${createFileName('NetPyNE_init_')}.py`;
-  const blob = new Blob([unescapeText(textData)], { type: 'text/plain;charset=utf-8' });
-  forceBlobDownload(blob, filename);
+export const downloadPythonResponse = (exportInfo) => {
+  const blob = new Blob([unescapeText(exportInfo.fileContent)], { type: 'text/plain;charset=utf-8' });
```

Comment:
this won't yet work since exportInfo is a string, the JSON parsing failed here because the file content has invalid characters. I added base64 encoding/decoding to fix this